### DISCUSSION TITLE
Those  initially private repos can't be found when editing pinned items after changed to public

### DISCUSSION THREAD

**Discussion by gingerhot** [community/community#3613] [Codespaces] [answered]
Those  initially private repos can't be found in the list when editing pinned items after changed them to public. If this is a bug or there's a cache need time to refresh?

**Comment by PranamBhat** [comment, marked as answer]
Hi @gingerhot 

This seems to be browser cache/cookies. I have tried this thing from my end, and it is working fine.

- Clear you browser history - all time by selecting all the checkboxes.
- Try refreshing (Ctrl+F5)

It will work. Thank you.

**Reply by gingerhot** [reply]
Yes, I tried refreshing several times and it works now. Thank you for your answer!
